src/Features/CSharp/Portable/EditAndContinue/SyntaxComparer.cs

```diff
@@ -61,7 +61,7 @@ public sealed override double GetDistance(SyntaxNode oldNode, SyntaxNode newNode
             return ComputeValueDistance(oldNode, newNode);
         }
 
-        internal static double ComputeValueDistance(SyntaxNode oldNode, SyntaxNode newNode)
+        internal static double ComputeValueDistance(SyntaxNode? oldNode, SyntaxNode? newNode)
```

Comment:
>SyntaxNode? oldNode, SyntaxNode? newNode [](start = 52, length = 40)

Are there callers that pass `null`?

Reply:
Yes there are several that pass `null` here. 

---
In reply to: [362869589](https://github.com/dotnet/roslyn/pull/40699#discussion_r362869589) [](ancestors = 362869589)